modules/nw-route-specific-annotations.adoc

```diff
@@ -16,95 +16,130 @@ To create an allow list with multiple source IPs or subnets, use a space-delimit
-|`router.openshift.io/haproxy.health.check.interval`| Sets the interval for the back-end health checks. (TimeUnits) | `ROUTER_BACKEND_CHECK_INTERVAL`
-|`haproxy.router.openshift.io/ip_allowlist`
-| Sets an allowlist for the route. The allowlist is a space-separated list of IP addresses and CIDR ranges for the approved source addresses. Requests from IP addresses that are not in the allowlist are dropped.
-
-The maximum number of IP addresses and CIDR ranges directly visible in the `haproxy.config` file is 61. [^2^] |
-
-|`haproxy.router.openshift.io/hsts_header` | Sets a Strict-Transport-Security header for the edge terminated or re-encrypt route. |
-|`haproxy.router.openshift.io/rewrite-target` | Sets the rewrite path of the request on the backend. |
-|`router.openshift.io/cookie-same-site` | Sets a value to restrict cookies. The values are:
+and "-". The default is the hashed internal key name for the route. 
+|
+
+|`haproxy.router.openshift.io/pod-concurrent-connections`
+|Sets the maximum number of connections that are allowed to a backing pod from a router. +
+Note: If there are multiple pods, each can have this many connections.  If you have multiple routers, there is no coordination among them, each may connect this many times. If not set, or set to 0, there is no limit. 
+|
+
+|`haproxy.router.openshift.io/rate-limit-connections`
+|Setting `'true'` or `'TRUE'` enables rate limiting functionality which is implemented through stick-tables on the specific backend per route. +
+Note: Using this annotation provides basic protection against distributed denial-of-service (DDoS) attacks. 
+|
+
+|`haproxy.router.openshift.io/rate-limit-connections.concurrent-tcp`
+|Limits the number of concurrent TCP connections made through the same source IP address. It accepts a numeric value. +
+Note: Using this annotation provides basic protection against distributed denial-of-service (DDoS) attacks.
+| 
+
+|`haproxy.router.openshift.io/rate-limit-connections.rate-http`
+|Limits the rate at which a client with the same source IP address can make HTTP requests. It accepts a numeric value.  +
+Note: Using this annotation provides basic protection against distributed denial-of-service (DDoS) attacks.
+|
+
+|`haproxy.router.openshift.io/rate-limit-connections.rate-tcp`
+|Limits the rate at which a client with the same source IP address can make TCP connections. It accepts a numeric value.  +
+Note: Using this annotation provides basic protection against distributed denial-of-service (DDoS) attacks.
+|
+
+|`haproxy.router.openshift.io/timeout` 
+|Sets a server-side timeout for the route. If not set, the default value is `60s`. (TimeUnits)
```

Comment:
The default value of  server-side timeout seems to be 30s(sorry, the https://docs.openshift.com isn't available, I will check it  again when the docs link is available)